<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bl4ckJackClass</class>
 <widget class="QMainWindow" name="bl4ckJackClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>476</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>476</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>bl4ckJack</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../bl4ckJack.gif</normaloff>../bl4ckJack.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Password Work Bench</string>
    </property>
    <widget class="QTableView" name="tblHash">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>581</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotalHashestxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Uncracked Hashes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPPStxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Passwords Per Second (PPS) </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCompletionTimetxt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Until Completion: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRecoveredPasswordstxt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Recovered Passwords: </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>591</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Recovered Passwords</string>
    </property>
    <widget class="QTableWidget" name="tblPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>571</width>
       <height>121</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; width: 60px; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Right click for more options.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <column>
      <property name="text">
       <string>Module</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Password</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hash</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="lblRecoveredPasswords">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCompletionTime">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPPS">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotalHashes">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Properties"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="action_Updates"/>
   </widget>
   <widget class="QMenu" name="menu_Control">
    <property name="title">
     <string>&amp;Control</string>
    </property>
    <addaction name="action_Start"/>
    <addaction name="separator"/>
    <addaction name="action_Pause"/>
    <addaction name="separator"/>
    <addaction name="actionS_top"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Control"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Updates">
   <property name="text">
    <string>&amp;Updates</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Properties">
   <property name="text">
    <string>&amp;Properties</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="actionS_top">
   <property name="text">
    <string>S&amp;top</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bl4ckjack.qrc"/>
 </resources>
 <connections/>
</ui>
